kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/options/PaymentMethodsForVendorValuesFinder.java

```diff
@@ -0,0 +1,26 @@
+package edu.arizona.kfs.fp.businessobject.options;
+
+import java.util.HashMap;
+import java.util.Map;
+import edu.arizona.kfs.sys.businessobject.options.PaymentMethodValuesFinder;
+
+
+/**
+ * This class returns list of payment method key value pairs.
+ * 
+ * Customization for UA: addition of "A" type for credit card payments.
+ * 
+ * @author jonathan
+ * @see org.kuali.kfs.fp.businessobject.options.PaymentMethodValuesFinder
```

Comment:
Please update comment as class was moved in KFS 6.0:
see org.kuali.kfs.sys.businessobject.options.PaymentMethodValuesFinder